gulpfile.js

```diff
@@ -77,6 +76,9 @@ gulp.task('styles', ['styles:fabricator', 'styles:toolkit']);
 
 
 // scripts
+var webpackCompiler = webpack(require('./webpack.config')(config));
```

Comment:
Can this be moved to the `var` block at the top, next to the `webpack` var delcaration? Just a matter style preference
.